### ISSUE TITLE
ariaRequired prop should render to aria-required attribute

### ISSUE THREAD

**Issue by shimedh** [facebook/draft-js#1696]
**BUG REPORT (AND POTENTIALLY A FEATURE REQUEST)**

There should be an ariaRequired prop for Editor control and it should render the provided id string to the aria-required attribute.

As of Draft 0.10.5 I don't see any attribute for ariaRequired.
https://github.com/facebook/draft-js/blob/master/src/component/base/DraftEditorProps.js

**Comment by EJLearner** [comment]
Also, aria-invalid is an important indicator that an input is in error. Will support for that be added?

**Comment by waynegerard** [comment]
Commented on the PR linked, but - is this still necessary? My understanding is that the required attribute as part of HTML5 supersedes the aria attribute, and support seems pretty widespread for that.
